assets/js/src/form_editor/components/editor.jsx

```diff
@@ -44,6 +44,11 @@ export default () => {
     (sel) => sel('mailpoet-form-editor').getSidebarOpened(),
     []
   );
+  const formBlocks = useSelect(
+    (sel) => sel('mailpoet-form-editor').getFormBlocks(),
```

Comment:
Nothing wrong here, I just wonder. Why did you have to make this change? What is wrong with `select('mailpoet-form-editor').getFormBlocks()`?

Reply:
Form Block didn't refresh when I clicked on undo without useSelect.